apps/vr-tests-react-components/src/stories/DatePickerCompat.stories.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react';
+import { Steps, StoryWright } from 'storywright';
+import { DatePicker as DatePickerBase, DateRangeType } from '@fluentui/react-datepicker-compat';
+import { storiesOf } from '@storybook/react';
+import { TestWrapperDecorator } from '../utilities/TestWrapperDecorator';
+import type { DatePickerProps } from '@fluentui/react-datepicker-compat';
+
+const DatePicker = (props: DatePickerProps) => {
+  const today = new Date('3/15/2023');
+  return (
+    <div style={{ width: '500px', height: '330px', padding: '10px' }}>
+      <DatePickerBase value={today} today={today} {...props} />
+    </div>
+  );
+};
+
+storiesOf('DatePicker Compat', module)
+  .addDecorator(TestWrapperDecorator)
+  .addDecorator(story => (
+    <StoryWright
+      steps={new Steps()
+        .snapshot('rest', { cropTo: '.testWrapper' })
+        .click('.datepicker-input')
+        .snapshot('opened', { cropTo: '.testWrapper' })
+        .end()}
+    >
+      {story()}
+    </StoryWright>
+  ))
+  .addStory('default', () => <DatePicker input={{ className: 'datepicker-input' }} />, {
+    includeHighContrast: true,
+    includeDarkMode: true,
+  })
+  .addStory(
+    'DateRange: day',
+    () => <DatePicker calendar={{ dateRangeType: DateRangeType.Day }} popover={{ open: true }} />,
+    {
+      includeDarkMode: true,
+      includeHighContrast: true,
+    },
+  )
+  .addStory(
+    'DateRange: week',
+    () => <DatePicker calendar={{ dateRangeType: DateRangeType.Week }} popover={{ open: true }} />,
+    {
+      includeDarkMode: true,
+      includeHighContrast: true,
+    },
+  )
+  .addStory(
+    'DateRange: work week',
+    () => <DatePicker calendar={{ dateRangeType: DateRangeType.WorkWeek }} popover={{ open: true }} />,
+    {
+      includeDarkMode: true,
+      includeHighContrast: true,
+    },
+  )
+  .addStory(
+    'DateRange: month',
+    () => <DatePicker calendar={{ dateRangeType: DateRangeType.Month }} popover={{ open: true }} />,
+    {
+      includeDarkMode: true,
+      includeHighContrast: true,
+    },
+  )
```

Comment:
nit: You could probably get by with including DarkMode/HighContrast on just one of these DateRange stories.